example/simple/generated_bindings.dart

```diff
@@ -87,51 +87,51 @@ class NativeLibrary {
 }
 
 typedef _c_sum = ffi.Int32 Function(
-  ffi.Int32 a,
-  ffi.Int32 b,
+  ffi.Int32,
```

Comment:
If we ever want to make those typedefs non-private (https://github.com/dart-lang/ffigen/issues/24), the arguments having names might actually be useful.

Even for code inspection right now the names could be useful. Maybe we should keep them.

Is there a reason we're removing them?

Reply:
No reason, I did this to not having to do conflict resolution for typedefs

Reply:
The typedefs use the same parameters so name conflict resolution will need to be updated